fresque/lib/views.py

```diff
@@ -98,3 +110,16 @@ def user_reviews(db, username):
             Reviewer.reviewer_name == username
         ).order_by(Review.date_start).all()
     return Result({"reviews": reviews})
+
+
+def create_git_repo(name, gitfolder):
+    ''' Create a git project based on the package information
+        and intialise a template spec file.
+    '''
+    gitrepo = os.path.join(gitfolder, '%s.git' % name)
+    if os.path.exists(gitrepo):
+        raise 'The project repo "%s" already exists' % name
```

Comment:
Instead of raising a bare string, can you raise an exception with that string as its message?  i.e.,

```python
raise IOError('The project repo "%s" already exists' % name)
```